packages/memcache-client/src/types.ts

```diff
@@ -43,6 +43,7 @@ export type MemcacheClientOptions = {
   server: ServerDefinition | SingleServerEntry | MultipleServerEntry | string;
   ignoreNotStored?: boolean;
   lifetime?: number;
+  noDelay?: boolean;
```

Comment:
Maybe initialize this in the Constructor of MemcacheClient in `client.ts`? Is this needed? i.e. something like 

`this.options.noDelay = options.noDelay || false;`

Reply:
This isn't necessary. Other boolean options like `ignoreNotStored` and `keepDangleSocket` aren't initialized, so I followed the existing pattern.

Reply:
OK Sounds good.